seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/src/test/java/org/apache/seatunnel/e2e/connector/http/HttpIT.java

```diff
@@ -245,6 +246,9 @@ public void testMultiTableHttp(TestContainer container)
                 mockServerClient.retrieveRecordedRequests(
                         request().withPath("/example/httpMultiTableContentSink").withMethod("POST"),
                         Format.JSON);
+        mockServerClient.clear(
```

Comment:
Sorry, I forgot to clear the records, which caused the run to fail. This test case will run multiple times. Please help add it

Reply:
Thank you very much @jackyyyyyssss ,Please cc

Reply:
LGTM